### DISCUSSION TITLE
Image is burning and smudging

### DISCUSSION THREAD

**Discussion by janemarie425** [arkypita/LaserGRBL#1169] [Q&A]
Hello,

I am very new to LaserGRBL. I recently purchased an Ortur Laser Master 15W.  When I prepare an image and send to the laser is is burning black the image and smudging.  Only one time did send an image to the laser and it more like carved it and didnt burn it. It finished brown and appeared deeper and did not smudge.  Please help.  Thanks kindly.
![image carve](https://user-images.githubusercontent.com/74572058/99318702-72366300-2825-11eb-8cad-a14e8fa30259.jpg)
![image burnt and smudge](https://user-images.githubusercontent.com/74572058/99318706-7793ad80-2825-11eb-859e-f279821fa232.jpg)

**Comment by StuartB4** [comment]
Start LaserGRBL and click connect. Check in Grbl configuration on the Grbl menu. and make sure Laser Mode is enabled. 
That is $32, set it to 1 if it's on 0. Check $30, it should be set to 255 ( or 1000).
If you have to change them click Write after and then Close. 

Then when you load an image and you get all the settings down the left side, adjust the Brightness and Contrast if you need to.
Then slide the White clip slider to the right until all the background interference around your image or text turns to pink. 

After you've done that and clicked Next make sure the S-Min setting is 0 and the and the S-Max is either 255 or 1000 to match whatever $30 is. Then you can adjust the power using S-Max for individual images. 

If Laser On in Target Image settings is set to M3 chnage it to M4.


![image](https://user-images.githubusercontent.com/16980026/99328992-21e6f180-2875-11eb-86cc-7929db7dc101.png)

![image](https://user-images.githubusercontent.com/16980026/99329703-adad4d80-2876-11eb-8d01-c785f0f6bb76.png)

**Comment by BeatSlayer** [comment]
Less power and slower.

**Comment by janemarie425** [comment]
Thank you so much!! I will give it a try. 

Kind regards,

Jane
